backend/dataall/api/Objects/Environment/resolvers.py

```diff
@@ -699,7 +699,51 @@ def get_pivot_role_template(context: Context, source, organizationUri=None):
             )
             raise e
 
+def get_cdk_exec_policy_template(context: Context, source, organizationUri=None):
+    from ....utils import Parameter
 
+    with context.engine.scoped_session() as session:
+        ResourcePolicy.check_user_resource_permission(
+            session=session,
+            username=context.username,
+            groups=context.groups,
+            resource_uri=organizationUri,
+            permission_name=permissions.GET_ORGANIZATION,
+        )
+        cdk_exec_policy_bucket = Parameter().get_parameter(
+            env=os.getenv('envname', 'local'), path='s3/resources_bucket_name'
+        )
+        cdk_exec_policy_bucket_key = Parameter().get_parameter(
+            env=os.getenv('envname', 'local'), path='s3/cdk_exec_policy_prefix'
+        )
```

Comment:
The parameter path needs to be prefixed, something like `parameter_path=f"/dataall/{os.getenv('envname', 'local')}/s3/...` otherwise the pivotRole won't have permissions to read it